ui/hooks/useGasFeeInputs.js

```diff
@@ -514,28 +485,28 @@ export function useGasFeeInputs(
     { value: ethBalance, fromNumericBase: 'hex' },
   );
 
+  const handleGasLimitOutOfBoundError = useCallback(() => {
+    if (gasErrors.gasLimit === GAS_FORM_ERRORS.GAS_LIMIT_OUT_OF_BOUNDS) {
+      const transactionGasLimit = hexToDecimal(transaction?.txParams?.gas);
```

Comment:
Do we want to call it `transactionGasLimitDec` to make it clearer and consistent with `minimumGasLimitDec`?

Reply:
++